plugins/opusCodec/src/OpusEncoder.h

```diff
@@ -0,0 +1,68 @@
+#ifndef OPUSENCODER_H
+#define OPUSENCODER_H
+#include <plugins/CodecPlugin.h>
+#include "opus/opus.h"
```

Comment:
Since this is an external library header please include it as `<opus/opus.h>` instead of using the local include style with double quotes

Reply:
This should be corrected in commit 24669c9